### ISSUE TITLE
Versions for some distributions are missing

### ISSUE THREAD

**Issue by vasti** [devops-works/binenv#168]
It looks like entries in versions file is missing for some of the supported distributions, at least in the latest at the moment: https://github.com/devops-works/binenv/blob/508cdc86648713394b0f369127224557ac1949c9/distributions/cache.json

_binenv_ itself is not there and if one follows the [quick install](https://github.com/devops-works/binenv#linux-bashzsh) instructions there are errors:
```
user@nix:~/tmp/binenv$ ./binenv install binenv
2021-10-16T22:25:24+03:00 WRN version for "binenv" not specified; using ""
2021-10-16T22:25:24+03:00 ERR unable to install "binenv" () error="unable to select latest stable version for \"binenv\": no stable version available. May be run 'binenv update binenv' ?"
```

The 'full' [install instructions](https://github.com/devops-works/binenv#install) instructs to specify a version of binenv to install, i.e.
```
./binenv install binenv <version>
```
This way installation works.

Is this expected behavior? It is somewhat inconvenient compared to running `binenv versions`.

**Comment by leucos** [comment]
Thanks for reporting this @vasti 
I think this issue has been solved yesterday (see #156 #157 #159 #162)
Could you try again please ?
Staring from `./binenv update` should be enough.

**Comment by vasti** [comment]
`binenv versions binenv` now shows versions (after `binenv update`).

@leucos thanks for the help.
